services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/RouteOptionsTest.java

```diff
@@ -511,8 +514,8 @@ public void routeOptionWithExcludedPoint() {
 
     List<String> queryParameters = Arrays.asList(url.getQuery().split("&"));
     assertTrue(
-      "url doesn't contain excluded point: " + url.toString(),
-      queryParameters.contains("exclude=point(1.0%202.0)")
+      "url doesn't contain excluded point: " + url,
+      queryParameters.contains("exclude=point%281.0+2.0%29")
```

Comment:
Does Direction API recognise an encoded point? Even if it does, I find this URL isn't easily readable. Is it possible to handle excluded points so that it stays like "point(1.0%202.0)"?

Reply:
At this point I don't want to over-complicate the setup by adding exclusions. I did start with that approach to not only exclude `(` and `)` but also `;` and `,` in correct places, to keep the URL as readable as possible. This starts to get very complicated if we dive into recursive checks for nested characters and just wasn't worth it in my opinion.

We can reconsider separately.

Reply:
I did check the point exclusion query and it doesn't thrown `InvalidArgument` on the backend, so I assume it works correctly but tagging @purew @mandeepsandhu for visibility, in case we should change the setup anywhere.